src/main/java/com/redhat/devtools/lsp4ij/server/definition/LanguageServerDefinition.java

```diff
@@ -158,6 +159,13 @@ public void setEnabled(boolean enabled, @Nullable Project project) {
         this.enabled = enabled;
     }
 
+    @ApiStatus.Internal
+    public void removeAssociations() {
```

Comment:
This is called by `LanguageServerRegistry#removeAssociationsFor()` to ensure that the server definition is also cleared before being updated. Otherwise there could be stale mappings in the server definition.